Output/UARTOut/arm/uart_serial.c

```diff
@@ -75,8 +75,30 @@
 #define UART_SFIFO  UART2_SFIFO
 #define UART_TWFIFO UART2_TWFIFO
 
-#define SIM_SCGC4_UART  SIM_SCGC4_UART2
-#define IRQ_UART_STATUS IRQ_UART2_STATUS
+#elif defined(_kii_v3_)
```

Comment:
So, it might be a good idea to convert these defines into structs instead.

I'd already started for kinetis here: https://github.com/kiibohd/controller/blob/master/Lib/kinetis.h#L2974

Reply:
I believe the uart defines in the atmel sam headers are already a struct. I just haven't copied them over yet. https://github.com/aethaniel/ExperimentalCore-sam/blob/master/module/tools/CMSIS_Devices/ATMEL/sam4s/include/component/uart.h#L41

Should I convert the kinetis code or leave it alone for now?